src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/UploadDocument.kt

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.CommandLineRunner
+import org.springframework.boot.SpringApplication
+import org.springframework.boot.WebApplicationType
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.test.context.ActiveProfiles
+import uk.gov.hmcts.reform.bulkscan.orchestrator.dm.DocumentManagementUploadService
+import java.io.File
+import java.nio.file.Files
+
+private val LOG = LoggerFactory.getLogger(UploadDocument::class.java)
+
+@SpringBootApplication
+@ActiveProfiles("nosb")
+class UploadDocument(
```

Comment:
Why we are adding more Kotlin test classes ?

Reply:
ok ill delete it.